ios/Video/RCTVideo.swift

```diff
@@ -1463,7 +1472,7 @@ class RCTVideo: UIView, RCTVideoPlayerViewControllerDelegate, RCTPlayerObserverH
     }
 
     func handleVolumeChange(player: AVPlayer, change: NSKeyValueObservedChange<Float>) {
-        guard let _player else { return }
+        guard let _player, onVolumeChange != nil else { return }
```

Comment:
not sure this is a real optmi, but OK

Reply:
I think that this can be faster as we don't access properties but not sure

Reply:
@KrzysztofMoch In fact a better optimization would be to avoid registering the callback in fact !

Reply:
Yes, I will do it in separate PR